api/api/src/main/scala/uk/ac/wellcome/platform/api/Router.scala

```diff
@@ -106,27 +106,30 @@ class Router(elasticClient: ElasticClient,
               )
             }
         }
-    })
+    }
 
   def singleWork(id: String, params: SingleWorkParams): Future[Route] =
-    transactFuture("GET /works/{workId}")({
+    transactFuture("GET /works/{workId}") {
       val index = params._index.map(Index(_)).getOrElse(elasticConfig.index)
       val includes = params.include.getOrElse(V2WorksIncludes())
       worksService
         .findWorkById(id)(index)
         .flatMap {
           case Right(Some(work: IdentifiedWork)) =>
-            val expandedPaths = params._expandPaths.getOrElse(Nil)
-            retrieveTree(index, work, expandedPaths).map {
-              workFound(work, _, includes)
-            }
+            if (includes.collection) {
```

Comment:
Would it make more sense to move this logic down to `DisplayWorkV2.apply` (or an overload thereof)? Feels uncomfortable to be doing a special case for collections here, but perhaps it's  actually simpler...

Reply:
The code inside the conditional calls off to `CollectionService`, which in turn callls off to elasticsearch, so not sure how the logic would exist in `display` unless we allowed passing in callbacks etc, which seems a bit messy.

Reply:
:+1: makes sense